### ISSUE TITLE
Can't generalize? 

### ISSUE THREAD

**Issue by himajara** [otavioarj/SAMit#1]
Hey Otavio, can you generalize this software to change every traffic? Thanks

**Comment by otavioarj** [comment]
Yes I can, but this is just a PoC with Scapy and MiTM... it's not a big deal anyway :P
